bbl/i18n/ru/BambuStudio_ru.po

```diff
@@ -5407,7 +5411,7 @@ msgstr "Максимальное количество недавних прое
 
 msgid "Maximum count of recent projects"
 msgstr ""
-"Максимальное количество проектов отображаемое в списке недавних проектов."
+"Максимальное количество проектов, отображаемое в списке недавних проектов."
```

Comment:
Please rephrase "Максимальное количество проектов, отображаемое в списке недавних проектов." to "Максимальное количество недавних проектов"

Reply:
It's okay here, as this is a clarification of msgid "Maximum recent projects"

Reply:
Why we need this clarification you are expert in UX? What if location will be changed?

Reply:
No matter where it is located, it will be everywhere "project list" (in the program interface it is in two places). I thought it would be clearer that way.